include/amici/backwardproblem.h

```diff
@@ -101,6 +101,13 @@ class BackwardProblem {
      */
     realtype getTnext(std::vector<realtype> const& troot, int iroot, int it);
 
+    /**
+     * @brief Computes the adjoint part of the gradient by solving a linear system.
+     *
+     * Valid only if the computed solution is a steady-state.
+     */
+    void computeIntegralForSteadyState();
```

Comment:
I will check it out. I need ExpData to generate the rhs of the problem or pass dJydy from the forward problem. That is why I place that in backwardproblem.h. The right-hand side of the problem is constructed for free in handleDataPointB.

Reply:
Ah, good point!
Yet, philosophy-wise, I'd prefer to include ExpData into SteadyStateProblem rather than mixing the steadystate and the adjoint part...